src/fcntl.rs

```diff
@@ -404,33 +404,33 @@ mod consts {
 mod consts {
     use libc::c_int;
```

Comment:
This needs to `use libc::self` here as well, same as for NetBSD. We just don't test on DragonFly. If you want to install the DragonFly target to test this, that'd be appreciated.